Include/ceval.h

```diff
@@ -7,6 +7,10 @@ extern "C" {
 
 /* Interface to random parts in ceval.c */
 
+/* PyObject_Call(), PyObject_CallFunction(), and PyObject_CallMethod()
+ * are documented and recommended API to call callable objects.
```

Comment:
I don't understand the "are documented" part, maybe remove it?

I suggest to be more explicit:

" PyEval_CallObjectWithKeywords() is kept for backward compatibility: PyObject_Call(), PyObject_CallFunction() and PyObject_CallMethod() are recommended to call a callable object."